utilities/common.py

```diff
@@ -1,11 +1,18 @@
 from enum import Enum
 import re
+import csv
 
 GERMLINE = 'Germline'
 SOMATIC = 'Somatic'
 MIXED = 'Mixed'
 SVs = {'INS', 'DEL', 'DUP', 'CNV', 'INV'}
 
+codeDict = {}
+#Read code table for molecular consequence data. System should be in first column, code should be in second, and the display
+#consequence name should be in the third column.
+with open('SOCodeTable.csv') as codeTable:
+    reader = csv.reader(codeTable)
+    codeDict = {rows[2]:[rows[0], rows[1]] for rows in reader}
```

Comment:
This isn't required. declare this as a constant instead of building a dictionary from a CSV file. Declare it exactly like [this](https://github.com/FHIR/genomics-operations/blob/main/app/common.py#L41). SOCodeTable.csv can be deleted